.sequelizerc

```diff
@@ -1,7 +1,7 @@
-import path from 'path';
-import '@babel/register';
```

Comment:
Why es6?

Reply:
@fridolinho. Thank you for this feedback but I think it should be in an ASK form because literally "Why es6?" is not very descriptive.

@shemaeric you are changing from es6 to es5, I kindly suggest you revert back to es6